src/push/Permission.js

```diff
@@ -62,13 +66,14 @@ export default class Permission {
             this._win.Notification &&
             this._win.Notification.requestPermission
         ) {
-            /* Chrome 23+ */
-            this._win.Notification
-                .requestPermission()
-                .then(resolve)
-                .catch(function() {
+            /* Safari 12+ */
```

Comment:
I would move the "Chrome 23+" comment and add it to this comment to show this block covers both Safari 12+ and Chrome 23+

Reply:
I don't think I understand. The line below (line 70) handles Safari 12's case and then (pun intended!) lines 71-75 handle Chrome 23+.

Reply:
I got it now. You can ignore this review then. See my other response for suggestions as to how these comments can be improved though.